sdks/python/container/Dockerfile

```diff
@@ -69,7 +69,7 @@ RUN ccache --set-config=sloppiness=file_macro && ccache --set-config=hash_dir=fa
 ####
 COPY target/apache-beam.tar.gz /opt/apache/beam/tars/
 RUN pip install --no-deps -v /opt/apache/beam/tars/apache-beam.tar.gz[gcp]
-RUN pip check
+RUN pip check || (echo "Container does not include required Beam dependencies or has conflicting dependencies. If Beam dependencies have changed, you need to regenerate base_image_requirements.txt files." && exit 1)
```

Comment:
Is it worth having a link to how to do this?

Reply:
There are pointers in the files, if users look them up: https://github.com/apache/beam/blob/master/sdks/python/container/py36/base_image_requirements.txt#L16-L22.

but we could make a wiki entry and link here.

Reply:
added a link